### DISCUSSION TITLE
setup.py

### DISCUSSION THREAD

**Discussion by ramisahib** [nullroute-commits/CMS-NBI-Client#2] [Q&A] [answered]
hi when i issue command python3 setup.py the output 

root@hala-ftth-automation:/var/www/html/CMS-NBI-Client# python3 setup.py
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help
how can resolve it

**Comment by nullroute-commits** [comment, marked as answer]
> hi when i issue command python3 setup.py the output 
> 
> root@hala-ftth-automation:/var/www/html/CMS-NBI-Client# python3 setup.py
> usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
>    or: setup.py --help [cmd1 cmd2 ...]
>    or: setup.py --help-commands
>    or: setup.py cmd --help
> how can resolve it

Are u following  a tutorial, as the setup.py file is to help build the package using pip?

Heres a stackoverflow q/a about it
http://stackoverflow.com/questions/1471994/ddg#1472014
